addons/addon-base-ui/packages/base-ui/src/helpers/notification.js

```diff
@@ -96,7 +96,7 @@ const toasterErrorOptions = {
   debug: false,
   newestOnTop: true,
   progressBar: true,
-  positionClass: 'toast-top-right',
+  positionClass: 'toast-bottom-right',
```

Comment:
Could you please add a screenshot for how the alert box look like for bottom-right?